### ISSUE TITLE
Error while creating an empty project

### ISSUE THREAD

**Issue by resilience-jychp** [bmarsh9/gapps#52]
When trying to add a empty project to a tenant I get an error.

This was done on a fresh instance (just created the tenant) :

![Capture d’écran du 2023-04-21 17-46-11](https://user-images.githubusercontent.com/113923302/233679654-838d34e6-7f9c-4e3c-9978-acf16af808f3.png)

Thanks for the job done so far, seems very promising project ...

**Comment by bmarsh9** [comment]
@resilience-jychp Oh yikes... Can you click on that Chat button in the bottom right and check what version you are running? Or if you know off hand

**Comment by resilience-jychp** [comment]
> @resilience-jychp Oh yikes... Can you click on that Chat button in the bottom right and check what version you are running? Or if you know off hand 

Can not check now but was a docker-compose on up to date main branch.

**Comment by bmarsh9** [comment]
@resilience-jychp Yep you are right. I think Ill just remove the option to create a empty project from that view. Instead, you can create your own framework and add controls. See the screenshots:

### 1.) Create framework under Controls > New Control
<img width="862" alt="g_framework" src="https://user-images.githubusercontent.com/26391921/233752654-88feb661-2026-488b-a001-fbab7570f9d2.PNG">

### 2.) Under Projects, select your added framework
<img width="349" alt="g_framework_2" src="https://user-images.githubusercontent.com/26391921/233752704-c77127d1-ab09-4ae4-b58e-6209c2702791.PNG">

**Comment by resilience-jychp** [comment]
Working fine.
Thank you for guidance.
